q2cli/handlers.py

```diff
@@ -504,6 +504,11 @@ def get_type(self):
                 return mapping['Float']
             else:
                 raise NotImplementedError()
+        # Specify 'Set'/'List' because 'Dict' will require more modifications
+        elif self.ast['name'] in ['Set', 'List']:
+            # n-`primitive`s can not be unioned, so ignore trailing values
+            field, *_ = self.ast['fields']
```

Comment:
This is guaranteed to be a single field, so you can just unpack the first value:
```python
field, = self.ast['fields']
```